op-program/client/interop/consolidate.go

```diff
@@ -72,6 +127,14 @@ func RunConsolidation(deps ConsolidateCheckDeps,
 	return eth.SuperRoot(consolidatedSuper), nil
 }
 
+func isInvalidMessageError(err error) bool {
+	// TODO: Create an error category for InvalidExecutingMessage errors in the cross package for easier maintenance.
```

Comment:
nit: Can we make sure we have a ticket logged for this please?

Reply:
yup, done.